src/lib/govuk/Radio.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
```

Comment:
This component is very necessary. If you check the HTML sample at https://design-system.service.gov.uk/components/radios/, it's so much code to copy. Wrapping in a Svelte component is quite similar to their Nunchucks system